modules/cluster/addons/kustomize/overlays/aws-ebs-csi-driver/kustomization.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
-- github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/alpha/?ref=v0.6.0
+- github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/alpha/?ref=release-0.9
```

Don't we want to keep using tags instead of branches?

```suggestion
- github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/alpha/?ref=v0.9.0
```

https://github.com/kubernetes-sigs/aws-ebs-csi-driver/tree/v0.9.0